config/system-test-driver.config.py

```diff
@@ -0,0 +1,122 @@
+#!/usr/bin/env python
```

Comment:
Yeah, I was hesitant to add the system test driver _"config"_ to the config directory for the same reasons... Where would you add the config/entrypoint?